src/headless.js

```diff
@@ -45,7 +43,7 @@ var window = {
     });
     window.timeouts.sort((x, y) => { return y.when - x.when });
   },
-  runEventLoop: function() {
+  runEventLoop: () => {
     // run forever until an exception stops this replay
     var iter = 0;
     while (!this.stopped) {
```

Comment:
Won't `this.` be affected by the change to arrow notation?

Reply:
Good catch.  Switch these to method notation.